pallets/gear/src/benchmarking/mod.rs

```diff
@@ -204,6 +204,7 @@ fn default_processor_context<T: Config>() -> ProcessorContext {
         reservation: 0,
         random_data: ([0u8; 32].to_vec(), 0),
         rent_cost: 0,
+        gas_to_value_multiplier: Default::default(),
```

Comment:
pls replace all related data to non-zero values because it's cheaper to write zeroes into memory so bench isn't that accurate